tueplots/figsizes.py

```diff
@@ -179,6 +179,31 @@ def neurips2021(
     )
 
 
+def neurips2022(
+    *,
+    rel_width=1.0,
+    nrows=1,
+    ncols=2,
+    constrained_layout=True,
+    tight_layout=False,
+    height_to_width_ratio=_GOLDEN_RATIO,
+):
+    """Neurips 2022 figure size."""
+
+    figsize = _from_base_in(
+        base_width_in=5.5,
```

Comment:
Uhmm I had another look at the 2021 style guide (https://media.neurips.cc/Conferences/NeurIPS2021/Styles/neurips_2021.pdf), and realized that the figsizes should _both_ be 5.5in. So the function you wrote is correct, but the existing figsize for neurips2021 is not :/

How do you feel about correcting this briefly? If you decide to tackle the suggestion about pointing to some base function in the other comment below, I think it would apply here too.
What do you think?

Reply:
Yes, I'm gonna fix it and implement the common `neurips_base` function for the default figure size.